lib/active_model_serializers.rb

```diff
@@ -18,11 +18,13 @@ def silence_warnings
 require 'active_model/serializer/version'
 
 require 'action_controller/serialization'
+require 'action_controller/serialization_test_case'
 ActiveSupport.on_load(:action_controller) do
   include ::ActionController::Serialization
   ActionDispatch::Reloader.to_prepare do
     ActiveModel::Serializer.serializers_cache.clear
   end
+  ActionController::TestCase.send(:include, ::ActionController::SerializationAssertions)
```

Comment:
Thanks for the review :heart: 

I try something in this line, but got no sucess :disappointed:  since Active Job defines the `ActiveJob::TestCase` in our case we do not want to define our `TestCase` class but mixes the helper in the `ActionController::TestCase`.

I followed the same idea we have in the in the [`ActionController::TestCase`](https://github.com/rails/rails/blob/e16afe61abd78c55f80752ca020b90d59ae1940f/actionpack/lib/action_controller/test_case.rb#L298-L300). I renamed/move the assertions module to a better place.

Let me know what you think